### ISSUE TITLE
NC15 Missing jquery.strengthify.min.js

### ISSUE THREAD

**Issue by MinIsMin** [nextcloud/server#13520]
### Steps to reproduce
1. Open /nextcloud/settings/user/security
2. See your logs

### Expected behaviour
No error should be thrown (like in 14.0.4)

### Actual behaviour
The following error is thrown:
`{"reqId":"123","level":3,"time":"2019-01-10T01:01:01+00:00","remoteAddr":"[ip]","user":"admin","app":"jsresourceloader","method":"GET","url":"\/nextcloud\/settings\/user\/security","message":"Could not find resource vendor\/strengthify\/jquery.strengthify.min.js to load","userAgent":"[useragent]","version":"15.0.2.0"}`

### Server configuration

**Operating system**: Ubuntu 18.04.1 LTS

**Web server:** Apache Apache/2.4.29

**PHP version:** PHP 7.2.10

**Nextcloud version:** 15.0.2.0

**Updated from an older Nextcloud/ownCloud or fresh install:** Update from 14.0.6

**Where did you install Nextcloud from:** Nextcloud zip (and upgrade)

**Signing status:** No errors have been found. (and no setup warnings)

**List of activated apps:** none (seems like a core issue)

<details>
<summary>Nextcloud configuration</summary>

```
{
    "system": {
        "htaccess.RewriteBase": "\/nextcloud",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "[domain]"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "15.0.2.0",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "theme": "",
        "loglevel": 2,
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "tls",
        "mail_smtpauth": 1,
        "updater.release.channel": "stable"
    }
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

**Comment by MorrisJobke** [comment]
@nextcloud/javascript @rullzer

**Comment by rigow** [comment]
In the meantime, this seems to cause KDEPIM not to be able to connect anymore with "An error occurred: There was a problem with the request. Invalid username/password (401)." Is there a hot fix that one can apply?

**Comment by MorrisJobke** [comment]
> In the meantime, this seems to cause KDEPIM not to be able to connect anymore with "An error occurred: There was a problem with the request. Invalid username/password (401)." Is there a hot fix that one can apply?

I guess there is something else a problem here but not a missing JS file.

**Comment by mko-x** [comment]
Fresh install, login via iOS app leads to 'Access denied. CSRF check failed' regardless user/password or app token.

Logs from Nextcloud:
```
[_jsresourceloader_] Error: Could not find resource vendor/strengthify/jquery.strengthify.min.js to load

GET /settings/user/security
from <ip> by <user> at 2019-01-24T20:34:13+00:00
```
Should #13546 work?

**Comment by MorrisJobke** [comment]
> Should #13546 work?

Yes

**Comment by kpleines** [comment]
Same problem on my 15.0.2.


Error | jsresourceloader | Could not find resource vendor/strengthify/jquery.strengthify.min.js to load
-- | -- | --

**Comment by MorrisJobke** [comment]
Fixed in 15.0.3 that is planned for Thursday.
